test/built-ins/RegExp/S15.10.2.12_A1_T1.js

```diff
@@ -60,7 +60,7 @@ if (i5.replace(/\s+/g, "") !== o5) {
 var i6 = "";
 for (var j = 6144; j < 7168; j++)
```

Comment:
This loop also needs to be updated now that the expected string no longer contains U+180E.

Reply:
oh sure, fixing it right now